src/twisted/python/compat.py

```diff
@@ -868,5 +894,6 @@ def _bytesRepr(bytestring):
     "intern",
     "unichr",
     "raw_input",
-    "_tokenize"
+    "_tokenize",
+    "toBytes"
```

Comment:
trailing commas are nice

Reply:
I would say that we should enforce trailing commas as a code style :) to reduce future diffs